include/compose.bash

```diff
@@ -216,6 +216,9 @@ cloudbreak:
         - CB_SMTP_SENDER_FROM=$CLOUDBREAK_SMTP_SENDER_FROM
         - CB_BAYWATCH_ENABLED=$CB_BAYWATCH_ENABLED
         - CB_BAYWATCH_EXTERN_LOCATION=$CB_BAYWATCH_EXTERN_LOCATION
+        - CB_CERT_DIR=/certs
```

Comment:
The java code has a different default value (/tmp/certs) - I can change the volume sharing of the cloudbreak container to /tmp/certs (I didn't want to use /tmp here) or I can change the default value in the java code to /certs (I didn't want to do that either because that doesn't really make sense if it isn't running in a docker container.